src/test/java/com/ge/predix/uaa/token/lib/FastTokenServiceTest.java

```diff
@@ -1,303 +0,0 @@
-    }
-
-    /**
-     * Tests that an token from the an untrusted issuer id throws an InvalidTokenException.
-     */
-    @Test(
-            expectedExceptions = InvalidTokenException.class,
-            expectedExceptionsMessageRegExp = ".*is not trusted because it is not in the configured list of trusted "
-                    + "issuers.")
-    public void testLoadAuthenticationWithUnstrustedIssuerId() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken("http://testzone1localhost:8080/uaa/oauth/token",
-                System.currentTimeMillis(), 60, false);
-        this.services.loadAuthentication(accessToken);
-    }
-
-    /**
-     * Tests that an expired token issues an InvalidTokenException.
-     */
-    @Test
-    public void testLoadAuthenticationWithExpiredToken() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken(System.currentTimeMillis() - 240000, 60);
-        try {
-            this.services.loadAuthentication(accessToken);
-            Assert.fail("Expected InvalidTokenException for expired token.");
-        } catch (InvalidTokenException e) {
-            System.out.println(e.getMessage());
-        }
-    }
-
-    /**
-     * Tests that an token that is valid for future use issues an InvalidTokenException.
-     */
-    @Test
-    public void testLoadAuthenticationWithFutureToken() throws Exception {
-        String accessToken = this.testTokenUtil.mockAccessToken(System.currentTimeMillis() + 240000, 60);
-        try {
-            this.services.loadAuthentication(accessToken);
-            Assert.fail("Expected InvalidTokenException for token issued in the future.");
-        } catch (InvalidTokenException e) {
-            System.out.println(e.getMessage());
-        }
-    }
-
-    /**
-     * Tests that null token issues an InvalidTokenException.
-     */
-    @Test(expectedExceptions = InvalidTokenException.class)
-    public void testLoadAuthenticationWithNullTokenString() throws Exception {
-        this.services.loadAuthentication("null");
-    }
-
-    /**
-     * Tests that empty token issues an InvalidTokenException.
-     */
-    @Test(expectedExceptions = InvalidTokenException.class, expectedExceptionsMessageRegExp = "Malformed Access Token")
-    public void testLoadAuthenticationWithEmptyToken() throws Exception {
-        this.services.loadAuthentication("");
-    }
-
-    @Test(expectedExceptions=IllegalArgumentException.class)
-    public void testWithNoTrustedIssuers() {
-        FastTokenServices  tokenService = new FastTokenServices();
-        tokenService.setRestTemplate(mockRestTemplate(false));
-        tokenService.loadAuthentication(this.testTokenUtil.mockAccessToken(60));
-    }
-
-    /**
-     * Tests that a tampered token issues an InvalidTokenException.
-     */
-    @Test(expectedExceptions = InvalidSignatureException.class)
-    public void testLoadAuthenticationWithTamperedToken() throws Exception {
```

Comment:
can we add missing tests from this class to src/test/java/com/ge/predix/uaa/token/lib/FastTokenServicesTest.java ?

Reply:
Added this test case.